### DISCUSSION TITLE
Apollo Save Tool v1.0.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps4#26] [Announcements]
### Added

* Encrypted save-games support
* Offline Activation with user-defined account IDs (`owners.xml`)
* Export ownership data to `.xml`

### Fixed

* Fixed inner file handling when save-game has sub-folders
* Fixed SaveWizard write codes

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps4/releases/tag/v1.0.0'>Apollo Save Tool v1.0.0</a>.</em>

**Comment by ghost** [comment]
Hi bucanero ty for everything you do for the Scene, i would really love to request a feature. which is a built in file manager inside apollo save tool.

**Reply by bucanero** [reply]
I still need to add more save-game related features, so this won't be a priority right now.

Meanwhile if you need a file manager, the best option available is PS4-Xplorer by Lapy.

**Comment by carnby77** [comment]
I already have the ps4 saves from my previous PS4 console on my thumbdrive. Now when i load apollo save tool 1.00, the saves in the thumb drive is describes as encrypted. So how do i get Apollo to decrypt the save file on the thumb drive and apply it to HDD saves?
Basically i want to transfer save files from a Stock PS4 to another PS4 which is jailbroken

P/S: My previous PS4 is not jailbroken. And the save file is different version (CUSA0XXA to CUSA0XXB)

**Reply by bucanero** [reply]
open the (Encrypted) USB save, and you should see an option to "copy save game to HDD". Select that option, and select "resign" when asked.
Now you should have the save on the HDD. You can go to the HDD saves menu, and open it, extract single files, decrypt it, etc. You could take the decrypted data files and swap them with the different version, but that depends that the game uses the same format in every version.

**Reply by bucanero** [reply]
Check this thread on reddit, it might give you some ideas to try: https://www.reddit.com/r/ps4homebrew/comments/shu33h/save_file_for_horizon/

**Reply by carnby77** [reply]
Yes!!! Finally made it. It was the keystone!! Thank You for making our life easier.

**Comment by Franseven** [comment]
Loving your work!! but i would love to see a pc version to maybe mass re-sign save files from a platinum save set for example to avoid the ps4 step to bring them on a ps5, that would be great, thanks for your efforts
